app/jobs/bulk_update_patients_from_pds_job.rb

```diff
@@ -8,7 +8,7 @@ class BulkUpdatePatientsFromPDSJob < ApplicationJob
   good_job_control_concurrency_with perform_limit: 1
 
   def perform
-    return unless Settings.pds.perform_jobs
+    return unless Settings.pds.perform_bulk_jobs
```

Comment:
What about `enqueue_bulk_updates`? Just because there's multiple jobs here, so it's not immediately clear which one it refers too.

Reply:
Yeah that's a good point, I've amended the commit and used your suggestion.